build_qtbase.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -x
+
+sudo sed -i -e 's/#deb-src/deb-src/' /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get -y build-dep libqt5gui5
+sudo apt-get -y build-dep qt4-x11
+sudo apt-get -y install libudev-dev libinput-dev libts-dev libxcb-xinerama0-dev libxcb-xinerama0
+
+QT_FOLDER=/home/pi/src/qt5
+INIT_FOLDER=$(pwd)
+
+if [ ! -d "$QT_FOLDER" ]; then
+  rm -rf $QT_FOLDER
+  mkdir $QT_FOLDER
+  cd $QT_FOLDER
+  git clone git://code.qt.io/qt/qtbase.git -b 5.9
+  cd qtbase
+  ./configure -release -opengl es2 -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/usr/bin/ -opensource -confirm-license -release -make libs -prefix /usr/local/qt5pi -extprefix $QT_FOLDER/qt5pi -no-use-gold-linker
+else
+  cd $QT_FOLDER/qtbase
+fi
+
+make -j 4
+make install
+cp -r /usr/share/fonts/truetype/dejavu/ $QT_FOLDER/qt5pi/lib/fonts
+tar -zcvf $INIT_FOLDER/qtbase.tar.gz $QT_FOLDER/qt5pi
```

Comment:
Just write it to the homedir.

Reply:
hmm. Is not it more logical to create an archive in the directory from which the script was called?

Reply:
No, because that makes it less consistent. You could accidentally fire it off from the wrong directory.